src/inspector_agent.cc

```diff
@@ -483,12 +498,32 @@ void AgentImpl::WaitForDisconnect() {
                            v8::ReadOnly).FromJust();                          \
   } while (0)
 
+static void AddCommandLineAPIMethod(
+    const v8::FunctionCallbackInfo<v8::Value>& info) {
+  auto env = Environment::GetCurrent(info);
+  v8::Local<v8::Context> context = env->context();
+
+  if (info.Length() != 2 || !info[0]->IsString() || !info[1]->IsFunction()) {
+    return env->ThrowTypeError("inspector.addCommandLineAPIMethod takes "
+        "exactly 2 arguments: a string and a function.");
+  }
+
+  v8::Local<v8::Object> console_api = env->inspector_console_api_object();
+  console_api->Set(context, info[0], info[1]);
```

Comment:
I hope I understood the ".FromJust" part correctly - I assume it's to make sure that a failing `Set()` actually causes an error..?